pkg/networkservice/core/eventchannel/monitor_connection_client.go

```diff
@@ -47,7 +47,7 @@ func NewMonitorConnectionMonitorConnectionsClient(ctx context.Context, eventCh <
 func (m *monitorConnectionMonitorConnectionsClient) Recv() (*networkservice.ConnectionEvent, error) {
 	select {
 	case <-m.ctx.Done():
-		return nil, m.ctx.Err()
+		return nil, errors.WithStack(m.ctx.Err())
```

Comment:
I think it should be "application context is done"
(and other places like `m.ctx.Err()`)